dashboard/src/components/Banner.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import "../styles/main.css";
+import { Nav, Navbar, Button, Form, NavDropdown, FormControl } from "react-bootstrap";
+
+const ROUTES = ["/", "/howto", "/leaderboard", "https://github.com/AndrooTheChen/REMi"];
+const TEXTS = ["Home", "How To", "Leaderboard", "Repository"];
+
+const Banner = () => (
+    <div class="container navbar-container mt-3">
+        {/* Navigation Bar */}
```

Comment:
Maybe name this component NavBar instead of Banner if it only contains a Navbar and you it's so important that people know it's a NavBar that you comment it in the component